frontend/src/modules/Catalog/index.js

```diff
@@ -0,0 +1,13 @@
+import { getModuleActiveStatus, ModuleNames } from 'utils';
+
+export const CatalogsModule = {
+  moduleDefinition: true,
```

Comment:
This is to avoid other possible object imports that can happen in the index files. We use this field to make sure that the object is a module definition. In the moduleUtils note that we are always filtering by this field. One example of object that is not a module definition is in Shares/index.js `export { ShareInboxList } from './components';`. Right now there is only this one, but I think it is good to have some control on the objects that we use then as modules.